CDCgov/prime-reportstream
# Updating the ORC mapping

This PR ...

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. Add data to ORC.1 and remove data from ORC.5 and make sure it show up as the status.
2. Remove data from ORC.17.1 and add data to ORC.17.9 and make sure it does not show up in the entering organization extension.
3. Make sure there is data in ORC.9. Ensure that you can find it under businessEvent in the FHIR mapping.

## Changes
- Removed ORC.17.9 from the entering organization extenstion
- Added ORC.1 as a backup for the status

## Checklist

### Testing
- [ Y] Tested locally?
- [ Y] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ N/A] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ N] Added tests?

### Process
- [ N] Are there licensing issues with any new dependencies introduced?
- [ Y] Includes a summary of what a code reviewer should test/verify?
- [ N] Updated the release notes?
- [ N] Database changes are submitted as a separate PR?
- [ N] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue 7570

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue N/A

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints? No
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints? No
- Does this change introduce new dependencies that need vetting? No 
- Does this change require changes to our infrastructure? No
- Does logging contain sensitive data? No
- Does this PR include or remove any sensitive information itself? No
